.tekton/pull-request.yaml

```diff
@@ -73,6 +73,22 @@ spec:
         workspaces:
           - name: source
             workspace: workspace
+      - name: check-partner-tasks
+        runAfter:
+          - build-container
+        taskSpec:
+          steps:
+            - name: check-task-structure
+              image: quay.io/redhat-appstudio/pull-request-builds:build-definitions-utils-{{revision}}
+              workingDir: $(workspaces.source.path)
+              script: |
+                #!/usr/bin/env bash
+                .tekton/scripts/check-partner-tasks.sh
```

Comment:
should be rather in ./hack/check-partner-tasks.sh -> so that user can run it manually.

Reply:
Done